.circleci/config.yml

```diff
@@ -115,20 +115,10 @@ jobs:
     parallelism: 1
     steps:
       - checkout # check out source code to working directory
-      - run: |
-          sudo mkdir -p /usr/local/bin
-          sudo chown -R circleci:circleci /usr/local/bin
-      - run: |
-          sudo mkdir -p /usr/local/lib/python3.6/site-packages
-          sudo chown -R circleci:circleci /usr/local/lib/python3.6/site-packages
-      - restore_cache:
-          key: pipdeps-{{ .Branch }}
-      - run: sudo pip install mkdocs mkdocs-material
-      - save_cache:
```

Comment:
Going to abandon caching for now since it was taking too long to get it working and it hasn't been speeding up builds in any meaningful way.